validator/testcases/javascript/predefinedentities.py

```diff
@@ -11,52 +12,88 @@
 
 # A list of identifiers and member values that may not be used.
 BANNED_IDENTIFIERS = {
-    u"newThread": "Creating threads from JavaScript is a common cause "
-                  "of crashes and is unsupported in recent versions of the platform",
-    u"processNextEvent": "Spinning the event loop with processNextEvent is a common "
-                         "cause of deadlocks, crashes, and other errors due to "
-                         "unintended reentrancy. Please use asynchronous callbacks "
-                         "instead wherever possible",
+    u"newThread": (
```

Comment:
On Tue, May 05, 2015 at 08:04:47AM -0700, Mathieu Agopian wrote:
>You don't need parens here to continue the strings.

I've added parens to most of these descriptions in the past, 
since they can optionally be tuples of strings rather than 
single strings, and it seemed sensible to use parens for all of 
them. I guess I don't really have an objection to removing them, 
though.